SECURITY.md

```diff
@@ -0,0 +1,20 @@
+# Security Policy
+
+If you have discovered a security vulnerability in this project, please report it
+privately. **Do not disclose it as a public issue.** This gives me time to work with you
+to fix the issue before public exposure, reducing the chance that the exploit will be
+used before a patch is released.
+
+Please submit information on the vulnerability as a
+[private report](https://github.com/libarchive/libarchive/security/advisories/new).
```

Comment:
Is this link correct? Shouldn't this be https://github.com/libfuse/libfuse/security/advisories/new instead?

Reply:
Oh, yikes, sorry! Fixed!